byudml/imputer/random_sampling_imputer.py

```diff
@@ -108,8 +109,14 @@ def produce(self, *, inputs: Inputs, timeout: float = None, iterations: int = No
                     # TODO: update column metadata?
             else:
                 self.logger.warning(
-                    'Cannot sample values to impute from column {} \'{}\', which has no known values'.format(i. col_name)
+                    'Cannot sample values to impute from column {} \'{}\', which has no known values'.format(
+                        i, col_name
+                    )
```

Comment:
This PR is looking good @erikhuck. How do you feel about replacing these warnings with a note in the primitive's docstring that says what behavior the primitive takes when a column with no known values is encountered? I wonder if that would be nice because it seems like it would make the primitive less chatty.